sdk/containerregistry/container-registry/swagger/containerregistry.json

```diff
@@ -779,9 +743,7 @@
     },
     "/acr/v1/{name}/_tags/{reference}": {
       "get": {
-        "tags": [
-          "Repository"
-        ],
+        "tags": ["Repository"],
         "description": "Get tag attributes by tag",
         "operationId": "ContainerRegistryRepository_GetTagAttributes",
```

Comment:
Would it be possible to rename this method `GetTagProperties()` since it now returns a `TagProperties` type?

Reply:
Done